shared-version.properties

```diff
@@ -1,4 +1,4 @@
 // Updating blackduck-common version here might necessitate an update to integration-common version in src/main/resources/create-gradle-airgap-script.ftl \
 // to ensure the integration-common versions in both airgap and non-airgap are the same. 
-gradle.ext.blackDuckCommonVersion='66.2.27'
+gradle.ext.blackDuckCommonVersion='66.2.29'
```

Comment:
We released 9.9.0 with blackduck-common version 66.2.28 -- looks like master in _two_ versions behind but really this MR updates Detect by one version only, which brings in https://github.com/blackducksoftware/blackduck-common/pull/439